readme.md

```diff
@@ -82,6 +86,10 @@ Use ActiveRecord to create a new `pbj` sandwich and make it belong to the studen
 
 ```rb
 # Your answer...
+geraldo = Student.create(name: "Geraldo")
+pbj = Sandwich.create(type: "pbj", student_id: 1)
```

Comment:
There's no assurance that `geraldo` has an ID of 1, so that probably won't work. In short, you should almost *never* need to type out a hard-coded id in code.

Instead, do one of the following:
```ruby
pbj = Sandwich.create(type: "pbj", student_id: geraldo.id)
pbj = Sandwich.create(type: "pbj", student: geraldo)
```